src/shop.js

```diff
@@ -436,14 +436,20 @@ function BinartaShopjs(checkpoint, deps) {
                 fsm.persist(ctx);
                 shop.localStorage.setItem('binartaJSPaymentProvider', provider)
             };
-            if (!fsm.getPaymentProvider())
-                if (shop.localStorage.getItem('binartaJSPaymentProvider'))
+            
+            if (!fsm.getPaymentProvider()) {
```

Comment:
It does not look like you have an automatic formatter installed as some of these curly braces you added do not have the spaces I would then expect. Anyway, while there is probably no formal code style defined it is safe to assume the code style is the one that is evident from looking at the code.

As a result control structure code blocks are not required to have curly braces and you do not need to add them when they are missing. In fact the rule here is to not add them when they are not needed and to when possible write your code such that they are not needed. This would force you to write functions to be called by the control structure branch you are on should you need more than 1 line. This control structure can then be appropriately named by you as to summarize the activity you expect to be carried out by that function. Thus documenting for other programmers what is supposed to happen and reducing their need to play compiler and parse the code. Obviously when doing that it is important that your named function actually aptly describes the code contained so people who parse it are not surprised to find it does something else.

Additionally. This commit is now heavily overloaded with your addition of curly braces and distracting from the rather small actual modification you needed to make. Thus making it harder to review this comment as I have to play "spot the difference" while "playing compiler" to see what has changed.